libbeat/docs/shared-path-config.asciidoc

```diff
@@ -0,0 +1,96 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Elastic Beats. Make sure you keep the
+//// descriptions here generic enough to work for all Beats that include
+//// this file. When using cross references, make sure that the cross
+//// references resolve correctly for any files that include this one.
+//// Use the appropriate variables defined in the index.asciidoc file to
+//// resolve Beat names: beatname_uc and beatname_lc.
+//// Use the following include to pull this content into a doc file:
+//// include::../../libbeat/docs/shared-path-config.asciidoc[]
+//// Make sure this content appears below a level 2 heading.
+//////////////////////////////////////////////////////////////////////////
+
+[[configuration-path]]
+=== Path
+
+The `path` section contains configuration options that define where the Beat
+is looking for certain types of files.
+
+Here is an example configuration:
+
+[source,yaml]
+------------------------------------------------------------------------------
+path:
+  home: /usr/share/beat
+  conf: /etc/beat
+  data: /var/lib/beat
+  logs: /var/log/
+------------------------------------------------------------------------------
+
+Note that it is possible to override these options via command line flags.
+Please see the <<directory-layout>> section for more details.
+
+==== Path Options
+
+You can specify the following options under the `path` section:
+
+===== home
+
+The home path for a {beatname_uc} installation is the default base path for all the
+other path settings as well for miscellaneous files coming with the distribution (e.g. the
+sample dashboards). If not set via a CLI flag or via the configuration file, the default
+for the home path is the location of the {beatname_uc} binary.
+
+Example:
+
+[source,yaml]
+------------------------------------------------------------------------------
+path:
+  home: /usr/share/beats
+------------------------------------------------------------------------------
+
+===== conf
+
+The configuration path for a {beatname_uc} installation is the default base path
+for the configuration files, including the main YAML configuration file and the
+Elasticsearch template file. If not set via a CLI flag, the default for the
+configuration path is the same as the home path.
+
+Example:
+
+[source,yaml]
+------------------------------------------------------------------------------
+path:
+  conf: /usr/share/beats/config
+------------------------------------------------------------------------------
+
+===== data
+
+The data path for a {beatname_uc} installation is the default base path for all
+the files in which {beatname_uc} needs to store its files. If not set via a CLI
```

Comment:
I think yo mean to say "for all the folders in which...." (instead of "for all the files in which").

This description still seems a bit vague. It would help if you could say a bit more about how the files are used. Maybe providing an example or two would help.